grafana-operator/grafana-operator
# Add support for arbitrary [Secure]JsonData

## Description
* Added `customJsonData` and `customSecureJsonData` fields to `GrafanaDashboard.spec.datasources[*]`
* Added example manifests with a non-standard plugin datasource

Note: Requires updating controller-gen to v0.6.2

## Relevant issues/tickets
Fixes https://github.com/grafana-operator/grafana-operator/issues/708

## Type of change




## Checklist


## Verification steps
1. Run
```
kubectl apply -f deploy/examples/GrafanaWithPlugin.yaml deploy/examples/datasources/Custom.yaml
kubectl get -o yaml configmap grafana-datasources
kubectl port-forward svc/grafana-service 3000:3000
```
2. Confirm data source correctly created at http://localhost:3000/datasources in browser
